### PR TITLE
GEODE-5011: ResultModel can add a table that takes a list of CliFunct…

### PR DESCRIPTION
**Pull request by jinmeiliao**
…ionResult

* added tests for TabularResultModel

Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [ ] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [ ] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

File: geode-core/src/main/java/org/apache/geode/management/internal/cli/result/model/ResultModel.java [hunk 1]
```diff
@@ -204,6 +204,20 @@ public TabularResultModel addTable(String namedSection) {
     return section;
   }
 
+  public TabularResultModel addTable(String namedSection, List<CliFunctionResult> functionResults,
+      boolean skipIgnored) {
+    Object model = sections.get(namedSection);
+    if (model != null) {
+      throw new IllegalStateException(
+          String.format("Section already exists. Can't overwrite it with this new content.",
```
Comment (jdeppe-pivotal) [inline geode-core/src/main/java/org/apache/geode/management/internal/cli/result/model/ResultModel.java]: This needs a format specifier - `%s`

File: geode-core/src/main/java/org/apache/geode/management/internal/cli/result/model/TabularResultModel.java [hunk 2]
```diff
@@ -68,4 +70,21 @@ public void addRow(String... values) {
       table.get(columnHeaders[i]).add(values[i]);
     }
   }
+
+  public boolean setContent(List<CliFunctionResult> functionResults, boolean skipIgnore) {
```
Comment (jdeppe-pivotal) [inline geode-core/src/main/java/org/apache/geode/management/internal/cli/result/model/TabularResultModel.java]: I don't think this is the right place for this logic - rather just pull this up to `ResultModel`

**Comment by jinmeiliao** [comment]
precheckin green

File: geode-core/src/main/java/org/apache/geode/management/internal/cli/result/model/ResultModel.java [hunk 3]
```diff
@@ -203,6 +203,29 @@ public TabularResultModel addTable(String namedSection) {
     return section;
   }
 
+  public TabularResultModel addTable(String namedSection, List<CliFunctionResult> functionResults,
```
Comment (jdeppe-pivotal) [inline geode-core/src/main/java/org/apache/geode/management/internal/cli/result/model/ResultModel.java]: I don't particularly like the side effect of status setting in this method but I'm not sure what a good alternative would be. I think for now, could the method name be changed to something like `addTableSettingStatus`?

**Pull request merged**